inst/rmarkdown/templates/mdpi/skeleton/skeleton.Rmd

```diff
@@ -80,33 +137,197 @@ abbreviations:
     long: Three letter acronym
   - short: LD 
     long: linear dichroism
-output: rticles::mdpi_article
+bibliography: mybibfile.bib
+appendix: appendix.tex
+endnotes: false
+output: 
+  rticles::mdpi_article:
+    latex_engine: pdflatex #pdflatex or XeLaTeX allowed
```

Comment:
I understand when `pdflatex` is set, the correct `classoption` should be set. This logic can be coded into the format function on `if (latex_engine == "pdfltax")`